content/docs/command-reference/remote/rename.md

```diff
@@ -0,0 +1,61 @@
+# remote rename
+
+Rename a data remote. This command affects DVC configuration files only, it does
```

Comment:
Just adding a quick suggestion to check whether I can commit to this branch:
```suggestion
Rename a DVC remote. This command affects DVC configuration files only, it does
```

Reply:
Thank you, I would synchronize this to the DVC repo.